__init__.py

```
@@ -337,53 +414,104 @@
+        LOG.info("setting the default eye color")
+        color_str = (message.data.get('color', None) or
+                     self.get_response('color.need'))
+        if color_str:
+            match = color_from_description(color_str)
+            if match is not None:
+                color_rgb = self._parse_to_rgb(match)
+                if color_rgb is not None:
+                    (r, g, b) = color_rgb.r, color_rgb.g, color_rgb.b
+                    self.settings["defaults"]["default_eye_color"] = [r, g, b]
+                    if self.settings.get("current_eye_color") != self.settings["defaults"]["default_eye_color"]:
+                        if self.ask_yesno('set.current.eye.color').lower() == "yes":
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)